code/game/objects/items/weapons/holosign.dm

```diff
@@ -58,6 +60,30 @@
 			qdel(H)
 		to_chat(user, "<span class='notice'>You clear all active holograms.</span>")
 
+/obj/item/holosign_creator/janitor
+	name = "Janitorial Holosign projector"
+	desc = "A handy-dandy holographic projector that displays a janitorial sign."
+	holosign_type = /obj/structure/holosign/wetsign
+	var/wet_timer = FALSE
```

Comment:
Call this `wet_enabled`. Suffixing a var with `timer` is used for `/datum/timer` holders